src/command/commands/write.rs

```diff
@@ -0,0 +1,94 @@
+//! Generic PUTting
+//!
+//! Open issues:
+//! * We should really GET first to
+//!   * know the typical content format (OK that could come from metadata too)
+//!   * know the last ETag the user saw (unless they force-push, aeh, force-put)
+//!   * For that, we need to be told of every request.
+//! * This should be available unconditionally (but, later when we auto-complete in commands, maybe
+//!   prefer resources that are annotated to be writable … oh, core-interfaces is still stalled)
+//! * Am I doing error handling right?
+//! * Maybe this should be more of an interactive edit command?
+
+use super::Command;
+use super::CommandHandler;
+// use super::CommandRegistry; // Look moo no handler!
```

Comment:
Yes, I took that thought and run with it. Removed in 0bdde235cc62b94c146dd323b70b86ff416865af